### DISCUSSION TITLE
How did you add NHL team logo/icon to led?

### DISCUSSION THREAD

**Discussion by reventhtv** [riffnshred/nhl-led-scoreboard#415] [Announcements]
Hi, This is an amazing project which can scale up to many sports. Wonderful job. 
I got an idea to do this for a cricket scoreboard where I want to display live scores using an API of my own or from the existing one's available. I want to know how did you fetch team logo to the led screen? Is it from the API?
Best Regards, 
Reventh

**Comment by falkyre** [comment]
Yes, it's something from the API that we are pulling an SVG of the team logo, then it gets converted to a format that we can display on the LED board.  We also do allow for overrides if you provide your own logo.  @riffnshred can provide more info on that as he wrote that portion of the code.

This is the code that will pull in the logo from the NHL API:  https://github.com/riffnshred/nhl-led-scoreboard/blob/9e9ed6f9b78c5025f8afac738742d84ff90ab85e/src/renderer/logos.py

**Comment by riffnshred** [comment]
@reventhtv Thanks for the kind word!

As @falkyre mentioned, we pull the SVG from the NHL API directly and convert them to PNG to the appropriate size, which gives the best image quality on those screens. You don't necessarily need to pull your logo from an API, you can download them from somewhere and host them on your board directly. I strongly recommend using SVG to then convert them to PNG (you can take our code and modify it to import the images from a folder on your raspberry pi).
